c2corg_api/views/sitemap.py

```diff
@@ -55,27 +59,11 @@ def collection_get(self):
                 ]
             }
         """
-        document_locales_per_type = DBSession. \
-            query(Document.type, func.count().label('count')). \
-            join(
-                DocumentLocale,
-                Document.document_id == DocumentLocale.document_id). \
-            filter(Document.type != USERPROFILE_TYPE). \
-            group_by(Document.type). \
-            all()
-
-        sitemaps = []
-        for doc_type, count in document_locales_per_type:
-            num_sitemaps = ceil(count / PAGES_PER_SITEMAP)
-            sitemaps_for_type = [
-                '/sitemaps/{}/{}'.format(doc_type, i)
-                for i in range(0, num_sitemaps)
-            ]
-            sitemaps.extend(sitemaps_for_type)
+        cache_key = _get_cache_key()
+        etag_cache(self.request, cache_key)
 
-        return {
-            'sitemaps': sitemaps
-        }
+        return cache_sitemap.get_or_create(
+            cache_key, _get_sitemap_index, expiration_time=-1)
```

Comment:
Not sure to have understood where and how cache keys are configured.
What if we finally want to set the cache expiration time to another value? (just curious again)